src/services/api/client.js

```diff
@@ -30,7 +26,8 @@ export default {
     return axios({
       method: method,
       url: apiUrl(requestPath),
-      data: data
-    });
+      data: data,
+      withCredentials: true
+    }).then(response => (response.data));
```

Comment:
tiny but I think our preferred convention would be

`.then((response) => response.data);`